components/org.wso2.carbon.identity.local.auth.smsotp.event.handler/src/test/java/org/wso2/carbon/identity/local/auth/smsotp/event/handler/notification/SMSNotificationHandlerTest.java

```diff
@@ -204,35 +241,4 @@ private Event constructSMSOTPEvent() {
 
         return smsOtpEvent;
     }
-
-    public class SMSNotificationHandlerExtended extends SMSNotificationHandler {
```

Comment:
For different testing scenarios the buildNotificaitonData function needed to return different values. The way to achieve this was to use a public static value return it from the function. This allows the value to can be set before the test and reset after.
Static variables for inner classes is not supported the supported java versions. Therefore moved the class outside.